tests/basictests/kafka/kafka-test.job.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: kafka-test
+  labels:
+    app: kafka-test
+spec:
+  template:
+    spec:
+      containers:
+      - name: kafka-test
+        image: quay.io/vpavlin/kafka-test
+        env:
+          - name: APP_FILE
+            value: app.sh
+          - name: NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+      restartPolicy: Never
+  backoffLimit: 1
```

Comment:
@vpavlin @crobby should this file be in the resource folder? We have all yaml files for other components there

Reply:
Moved